src/components/CreateList.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div @click="closeForm" class="fullscreen">
+    <form v-on:submit.prevent="">
+      <input
+        v-focus
+        type="text"
+        v-model="listTitle"
+        placeholder="Ente the list name"
+        @keyup.enter="onClick"
```

Comment:
Please remove this **keyup** event listener and just use **submit** event on form